Projects/UOContent/Spells/First/NightSight.cs

```diff
@@ -58,7 +58,7 @@ protected override void OnTarget(Mobile from, object targeted)
                     }
                     else
                     {
-                        from.SendMessage("{0} already have nightsight.", from == targ ? "You" : "They");
+                        from.SendMessage($"{(from == targ ? "You" : "They")} already have nightsight.");
```

Comment:
We don't have to do it here but basic conditionals should be changed to if/else with static messages.

Reply:
Done, also for other occurrencies.